src/hooks/index.js

```diff
@@ -318,10 +318,10 @@ exports.restrictToCurrentSchool = context => getUser(context).then((user) => {
 	if (params.route && params.route.schoolId && params.route.schoolId !== currentSchoolId) {
 		throw new Forbidden('You do not have valid permissions to access this.');
 	}
-	if (context.method === 'get' || context.method === 'find') {
+	if (context.method === 'get' || context.method === 'find' || context.method === 'remove') {
 		if (params.query.schoolId === undefined) {
 			params.query.schoolId = user.schoolId;
-		} else if (params.query.schoolId !== currentSchoolId) {
+		} else if (params.query.schoolId.toString() !== currentSchoolId) {
```

Comment:
I expect this breaks a bunch of routes. Why is this necessary?

Reply:
good question...